src/AppPrintOnly.test.tsx

```diff
@@ -300,6 +302,39 @@ it('VxPrintOnly flow', async () => {
   // Check Printed Ballots Count
   getAllByTextWithMarkup('Ballots Printed: 3')
 
+  // Blank Ballot should print
+  card.insertCard(
+    createVoterCard(),
+    encodeBallot({
+      election: electionSample,
+      ballotId: 'test-ballot-id',
+      ballotStyle: electionSample.ballotStyles[0],
+      precinct: electionSample.precincts[0],
+      votes: {},
+      isTestBallot: true,
+      ballotType: BallotType.Standard,
+    })
+  )
+
+  // Show Printing Ballot screen
+  await advanceTimersAndPromises()
+  getByText('Printing your official ballot')
+
+  expect(getAllByTextWithMarkup('[no selection]')).toHaveLength(20)
```

Comment:
```suggestion
  // All contests should have no selection
  expect(getAllByTextWithMarkup('[no selection]')).toHaveLength(20)
```

Reply:
Wait… didn't this get changed to "no selection" without the brackets? Maybe this code is on the `electionguard` branch?

Reply:
This is on `master`, I don't think you've updated the ballot on `master`, right?